### PR TITLE
Remove PATENTS text from a few straggler files

### PR DESCRIPTION
**Pull request by sagar0**
Remove PATENTS related wording from a few stragglers which still reference the old PATENTS file.

### CODE REVIEW

File: port/win/win_jemalloc.cc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 //  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
-//  This source code is licensed under the BSD-style license found in the
-//  LICENSE file in the root directory of this source tree. An additional grant
-//  of patent rights can be found in the PATENTS file in the same directory.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
```
Comment (pombredanne) [inline port/win/win_jemalloc.cc]: Was not that file under a BSD from LevelDB originally?

**Comment by pombredanne** [comment]
Thanks!
I wonder if that was not kept back because of the LevelDB and/or International Business Machines extra copyrights?

File: port/win/win_jemalloc.cc (hunk 1 above)
Comment (sagar0) [reply port/win/win_jemalloc.cc]: I am not sure if this file came from LevelDB. Even if it did, we still have the BSD-related text on lines 6-8, similar to all LevelDB-imported files.

Comment (siying) [reply port/win/win_jemalloc.cc]: I believe these are mostly from relatively new files, and from open-source community. They probably tested with older versions in their branches, and forgot to update the files. This specific file was from a pull request of yuslepukhin. All his other files followed the new header. So I believe it is mistake.

**Comment by sagar0** [comment]
> I wonder if that was not kept back because of the LevelDB and/or International Business Machines extra copyrights?

The PPC related files were contributed by IBM, and I believe the reason the old license text is still in these files is because both RocksDB-license-change and IBM-ppc-crc-contribution happened around the same time (or at least were in review during an intersecting period). The ppc files could have originally been created by just copy-pasting the standard old template header from other files at that time.

**Pull request closed without merging**